overlord/state/state.go

```diff
@@ -171,6 +172,8 @@ type marshalledState struct {
 	LastTaskId   int `json:"last-task-id"`
 	LastLaneId   int `json:"last-lane-id"`
 	LastNoticeId int `json:"last-notice-id"`
+
+	NoticeLastDate time.Time `json:"notice-last-date,omitempty"`
```

Comment:
Should this be changed to `NoticeLastTimestamp` with `"notice-last-timestamp"` as well?

Reply:
Yes. Done.